geodiff/src/changesetvarint.h

```diff
@@ -217,7 +217,7 @@ static u8 sqlite3GetVarint( const unsigned char *p, u64 *v )
 ** single-byte case.  All code should use the MACRO version as
 ** this function assumes the single-byte case has already been handled.
 */
-static u8 sqlite3GetVarint32( const unsigned char *p, u32 *v )
+static inline u8 sqlite3GetVarint32( const unsigned char *p, u32 *v )
```

Comment:
both static and inline?

Reply:
hmm, I forgot why I added it there I think as it is static, in some include these was this function unsused or so